delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
```

Comment:
The current implementation does not include any unit tests for the delete functionality. Consider creating tests to verify that books are deleted correctly and handle cases where deletion might fail.

```php
use PHPUnit\Framework\TestCase;

class BookRepositoryTest extends TestCase {
    public function testDeleteBook() {
        $mockConnection = $this->createMock(mysqli::class);
        $mockStmt = $this->createMock(mysqli_stmt::class);

        $mockConnection->method('prepare')->willReturn($mockStmt);
        $mockStmt->expects($this->once())->method('bind_param')->with($this->equalTo('i'), $this->anything());
        $mockStmt->method('execute')->willReturn(true);

        $repository = new BookRepository($mockConnection);
        $this->assertTrue($repository->deleteBook(1));
    }
}
```